gradle.properties

```diff
@@ -37,6 +37,10 @@ gtestRevision=07f4869221012b16b7f9ee685d94856e1fc9f361
 
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 org.gradle.workers.max=4
+slackApiVersion=1.2.0
+ktorVersion=1.2.1
+shadowVersion=5.1.0
+metadataVersion=0.0.1-dev-10
```

Comment:
It should be used also here https://github.com/JetBrains/kotlin-native/blob/0fc0cb221ea5b45b85d1a53c030d889c9684d65e/Interop/StubGenerator/build.gradle#L40